packages/components/src/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,164 @@
+host*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+:host {
+  --scale-tooltip-border-radius: var(--scl-radius-4);
+  --scale-tooltip-background-color: var(--scl-color-grey-60);
+  --scale-tooltip-color: var(--scl-color-white);
+  --scale-tooltip-font-weight: var(--scl-font-weight-regular);
+  --scale-tooltip-font-size: var(--scl-font-size-16);
+  --scale-tooltip-line-height: 1.4;
+  --scale-tooltip-padding: var(--scl-spacing-2) var(--scl-spacing-8);
+  --scale-tooltip-arrow-size: 5px;
+  --scale-tooltip-arrow-start-end-offset: var(--scl-spacing-8);
+
+  --scale-tooltip-max-width: 20rem;
+  --scale-tooltip-hide-delay: 0s;
+  --scale-tooltip-hide-duration: 0.125s;
+  --scale-tooltip-hide-timing-function: ease;
+  --scale-tooltip-show-duration: 0.125s;
+  --scale-tooltip-show-timing-function: ease;
+  --scale-tooltip-z-index: var(--scl-z-index-70);
```

Comment:
We need to rename these following the conventions 🤓 

https://gist.github.com/acstll/13b005c6c75e0e550c656b3fdaecb965#naming-convention-for-component-scoped-css-variables